cvat-cli/requirements/base.txt

```diff
@@ -1,3 +1,3 @@
 cvat-sdk~=2.15.0
 Pillow>=10.3.0
-setuptools>=65.5.1 # not directly required, pinned by Snyk to avoid a vulnerability
+setuptools>=70.0.0
```

Comment:
As far as I can tell, setuptools is still not directly required, so why remove the comment?

Reply:
Misunderstood the comment, returned it